Library/src/Systems/Users/UserSystem.cpp

```diff
@@ -581,6 +581,27 @@ void UserSystem::GetAgoraUserToken(const AgoraUserTokenParams& Params, UserToken
 	static_cast<chs_aggregation::ExternalServiceProxyApi*>(ExternalServiceProxyApi)->serviceProxyPost(TokenInfo, ResponseHandler);
 }
 
+void UserSystem::ResendVerificationEmail(const csp::common::String& InEmail,
+										 const csp::common::Optional<csp::common::String>& InRedirectUrl,
+										 NullResultCallback Callback)
+{
+	csp::common::String Email  = InEmail;
```

Comment:
These two can be const

Reply:
They can, but it also doesn't make a difference when it gets compiled.